Example/SampleSporify.xcodeproj/xcuserdata/daniel.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -7,7 +7,7 @@
 		<key>SampleSporify.xcscheme</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>2</integer>
+			<integer>1</integer>
```

Comment:
I am almost certain these two files do not have to be in the repo but decided to commit them anyways as I am not sure whether this makes a difference or not :/